libsysbox/sysbox/sysbox.go

```diff
@@ -59,16 +69,46 @@ func checkUnprivilegedUserns() error {
 		}
 
 		var val int
-		fmt.Sscanf(string(b), "%d", &val)
+		_, err = fmt.Sscanf(string(b), "%d", &val)
+		if err != nil {
+			return err
+		}
 
 		if val != 1 {
 			return fmt.Errorf("kernel is not configured to allow unprivileged users to create namespaces: %s: want 1, have %d", path, val)
 		}
+
+	case "fedora", "centos", "redhat":
+		path := "/proc/sys/user/max_user_namespaces"
```

Comment:
Are you sure `max_user_namespaces` has the same meaning as `unprivileged_userns_clone` ? It sounds more like a max limit rather than "allow non-root users to create user-namespaces". Just checking.

Reply:
Yep, I'm sure, this is the variable utilized by all non-debian-based distros.